requests/CreateProductRequest.go

```diff
@@ -0,0 +1,27 @@
+package requests
+
+import (
+	"fmt"
+	"github.com/gofiber/fiber/v2"
+	u "go-fiber-example/m/v2/utils"
+)
+
+type CreateProductRequest struct {
+	Code  string
+	Price string
+}
+
+func ValidateCreateProductRequest(ctx *fiber.Ctx) CreateProductRequest {
+	request := new(CreateProductRequest)
+	ctx.BodyParser(request)
+
+	locale := ctx.Get("Accept-Language")
+	
+	ve := u.Check("Code", request.Code, []string{u.NotNull}, locale)
+	ve = u.Check("Price", request.Price, []string{fmt.Sprintf("%s:%d", u.MinLength, 6)}, locale)
+
+	if len(ve) > 0 {
+		panic(ctx.Status(fiber.StatusUnprocessableEntity).JSON(ve))
```

Comment:
don't use panic to handle error, you should return ve or converting ve to error as returning value